### DISCUSSION TITLE
Higher resolution thumbnail for Media Picker V3 to prevent blurry preview

### DISCUSSION THREAD

**Discussion by mcl-sz** [umbraco/Umbraco-CMS#14695] [Features and ideas]
While Media Picker V3's crop function works fine, the lower resolution thumbnail used as a preview may distort the final appearance on the page.

Because `/umbraco/backoffice/UmbracoApi/Images/GetBigThumbnail` is used as a preview for the crop, the image will become blurry if the size exceeds 500 pixels or if a specific part of an image is cut.

Contentmanagers sometimes think that the end result will be blurry, while this is almost never the case.

Perhaps a slightly higher resolution could be used as the preview image so that it becomes less blurry.

**Comment by bergmania** [comment]
Not sure if the client knows how big the image needs to be, but I guess the endpoint could allow multiple sizes.

By the end of the day it is also a trade off, of how many sizes of the image that is cached.
